irods/library/irods_avu

```diff
@@ -64,17 +64,18 @@ options:
   state:
     description: >
       This is the desired state to achieve. "set" means a single AVU is
-      attached with this attribute. "present" means that the AVU will be
-      attached if there isn't already an AVU with the same attribute and
-      value. Any others will be removed. "absent" means that an AVU with
-      the provided attribute and value will be removed. "add" means that an AVU will be added if it already exists, elase it will be set.
+      attached with this attribute. "present" means that an AVU will be added as long as there isn't already one with the same attribute.
+      "absent" means that an AVU with the provided attribute and value will be removed. 
+        "add" means that an AVU will be added if there isn't already one with the same attribute, value, and unit.
```

Comment:
Please fix indentation and excessive line length.

Reply:
This is still janky. Line 71 is indented more than the others. Line 68 is really short, while lines 69 and 71 are really long. Please clean this up. Maybe wrap a column 80 or 100.